android/KMEA/app/src/main/java/com/tavultesoft/kmea/KeyboardPickerActivity.java

```diff
@@ -334,6 +334,11 @@ protected static boolean updateLexicalModelsList(Context context, ArrayList<Hash
   }
 
   private static void setSelection(int position) {
+    // Poison pill for Sentry integration. Do not leave for production
```

Comment:
This will be immediately removed after I see how the Sentry events are handled with the nightly build.

Reply:
Could you add a temp menu item in settings instead? I am not all that keen on even temporary errors like this ending up in a published build.

Reply:
ok. I've moved the intentional crash to a temporary menu item "Crash"